app/src/main/res/values/strings.xml

```diff
@@ -151,7 +151,7 @@
     <string name="settings_hosts_summary">Systemless hosts support for Adblock apps</string>
     <string name="settings_hosts_toast">Added systemless hosts module</string>
     <string name="settings_app_name_hint">New name</string>
-    <string name="settings_app_name_helper">App will be repackaged to this name</string>
```

Comment:
Unmarked noun error.

Reply:
Also still sensible and understandable.

Reply:
I'll agree this is a bit clumsy, and your proposed change for this one is good.

Reply:
This should still say "App will" actually, since there's very little space in that dialog.

Reply:
I moved a shorted explanation up to the "New name" and removed the helper.

Reply:
Removing a string is not your decision to make. You've broken the build.

Reply:
Sorry for that. Made it an empty string now.
Between broken English, and strings out of bounds in English or in translations, it seems to me to be the better option.